corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -13,11 +13,18 @@
         }
     </style>
 {% endblock %}
+{% block js %}{{ block.super }}
```

Comment:
Looking at extending the reporting templates, it seems like that would bring in a lot of ajax report loading functionality (beyond group filtering). If that's something that we are moving towards (reloading Project Performance Dashboard via ajax), I can definitely work on that.